tests/playwright/src/index.ts

```diff
@@ -0,0 +1,60 @@
+/**********************************************************************
+ * Copyright (C) 2023-2024 Red Hat, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2024 Red Hat, Inc.
```

it's a new file, was not there in 2023